src/platforms/unity/configuration/offline-caching.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: Offline Caching
+description: "Learn how the SDK handles offline caching."
+sidebar_order: 200
+---
+
+Offline caching is enabled by default. When a device is unable to send an event it is cached and we send a report once we receive another event. The SKD uses `Application.persistentDataPath` as base directory for caching. If you change this path take note that it should persist between sessions to capture events that would be lost otherwise.
```

Comment:
`l`: some wording suggestions.
```suggestion
Offline caching is enabled by default. When a device cannot send an event, it is cached, and the SDK sends a report once we receive another event. The SKD uses `Application.persistentDataPath` as the base directory for caching. If you change this path, take note that it should persist between sessions to capture events that would be lost otherwise.
```